src/Controller/Admin/CategoryController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Category;
+use Doctrine\ORM\EntityManagerInterface;
+use FOS\RestBundle\View\ViewHandlerInterface;
+use Sulu\Bundle\CategoryBundle\Category\CategoryManagerInterface;
+use Sulu\Bundle\CategoryBundle\Controller\CategoryController as SuluCategoryController;
+use Sulu\Bundle\CategoryBundle\Entity\CategoryRepositoryInterface;
+use Sulu\Component\Rest\ListBuilder\Doctrine\DoctrineListBuilderFactoryInterface;
+use Sulu\Component\Rest\ListBuilder\Metadata\FieldDescriptorFactoryInterface;
+use Sulu\Component\Rest\RestHelperInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class CategoryController extends SuluCategoryController
```

Comment:
Maybe its would better to use a `ResponseListener` todo that so the dev don't need to know about the requirements from the SuluCategoryController and the things like unserialize and serialize which are not so clean will be the same, but at the current architecture I see no other way.